.gitignore

```diff
@@ -1,4 +1,4 @@
 .DS_Store
-
```

Comment:
Why did you Change the `.gitignore` file?

Reply:
To exclude *.iml file (specific to Intellij), is that a problem?

Reply:
I don't think there is need to add a new exclude to the gitignore file for now.

Reply:
Ok @MartinSchoeler, i have reverted the commit